sdk/trace/provider.go

```diff
@@ -241,10 +241,7 @@ func (p *TracerProvider) Shutdown(ctx context.Context) error {
 	if !ok {
 		return fmt.Errorf("failed to load span processors")
 	}
-	if len(spss) == 0 {
```

Comment:
Last question: why are you removing this? If there are no processors, there's nothing to shutdown.

Reply:
The `for` loop already "checks" for empty slice, no need to explicitly do that?